src/components/app-navigator/TopTabNavigator.tsx

```diff
@@ -1,60 +1,77 @@
+import { ComponentType, ReactNode } from 'react'
+
 import { createMaterialTopTabNavigator } from '@react-navigation/material-top-tabs'
-import { View } from 'react-native'
 
 import { TopTabBar } from 'app/components/top-tab-bar'
 
 const Tab = createMaterialTopTabNavigator()
 
-type ScreenInfo = {
+const screenOptions = {
```

Comment:
does it make sense to get the colors and styles from a useStyles?

Reply:
I would say yes